Description: fix 'undefined symbol: sigar_skip_token' problem as suggested
 in upstream's issue tracker: https://github.com/hyperic/sigar/issues/60
Author: Andrius Merkys <merkys@debian.org>
--- a/bindings/java/hyperic_jni/jni-build.xml
+++ b/bindings/java/hyperic_jni/jni-build.xml
@@ -305,6 +305,7 @@
         <compilerarg value="-Wall"/>
         <compilerarg value="-Werror" if="jni.werror"/>
         <compilerarg value="${jni.gccm}" if="jni.gccm"/>
+        <compilerarg value="-fgnu89-inline"/>
         <defineset>
           <define name="${jni.define.name}_LINUX"/>
           <define name="_REENTRANT"/>
